src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/infrastructure/clients/GovNotifyNotificationSender.java

```diff
@@ -21,21 +24,28 @@ public class GovNotifyNotificationSender implements NotificationSender {
     private final int deduplicateSendsWithinSeconds;
     private final RetryableNotificationClient notificationClient;
```

Comment:
I know it will work without a qualifier but to keep it readable, can you add qualifier for the above client as well.